models/aeif_cond_alpha.h

```diff
@@ -105,6 +105,14 @@ For implementation details see the
 
 See also [1]_.
 
+.. admonition::
+   Under some conditions, the exponential function inside the numeric solver
+   routine for this model can cause a numerical instability. If the parameter
+   ``I_soma_max`` is set to a non-zero value, the total somatic current will
+   be evaluated at each solver timestep and its absolute value will be limited
+   to this value. Please note that setting ``I_soma_max`` could subtly change
+   the dynamics of the model.
+
```

Comment:
I'd want to work out this comment a bit more. For one, setting `I_spike_max` will change behavior (although it might not be noticeable). I would also focus more on the neurobiology than the numerics (unnaturally large current up to several Ampere) and compare to the reference implementation (https://brian2.readthedocs.io/en/stable/examples/frompapers.Brette_Gerstner_2005.html): They use forward Euler with a fixed time step. Now assume that at the beginning of a timestep the membrane potential is immediately below the threshold for calling a spike, which is $V_T+5\Delta_T$ in their example. Then, `I_spike` in their implementation can never exceed $g_L\Delta_T e^5=30\text{nS}\times2\text{mV}\times e^5\approx 8900\text{pA}$ if my math is right. It could be useful to mention this to give an indication of plausible choices. Obviously, even more interesting would be to have upper bounds on such a current from neurobiological sources, but I have none at hand right now.